scripts/clean_tests.py

```
@@ -0,0 +1,353 @@
+    "pytest_collection_modifyitems",
+    "pytest_generate_tests",
+    "pytest_runtest_makereport",
+    "pytest_runtest_protocol",
+    "pytest_sessionstart",
+    "pytest_sessionfinish",
+    "pytest_unconfigure",
+}
+
+
+def run_cmd(cmd):
+    result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.15.11)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)